ibis/expr/operations/core.py

```diff
@@ -67,10 +66,6 @@ def equals(self, other):
     def inputs(self):
         return self.args
 
-    @property
-    def exprs(self):
```

Comment:
This is a public API, so I don't think we can remove it yet. I'll update the PR.

Reply:
Or we can mark this PR as 4.0 without preserving backward compat APIs.

Reply:
pls just deprecate removing apis is causing breakages

Reply:
That's the idea!